### PR TITLE
MGMT-7786: remove openshift_version in kube-api flow

### PR DESCRIPTION
**Pull request by danielerez**
kube-api flow requires only the OS images list (instead of openshift_versions map).
Since assisted-service is using defaults values if OS_IMAGES is missing,
pass empty objects for OPENSHIFT_VERSIONS and RELEASE_IMAGES.

### CODE REVIEW

**Comment by danielerez** [comment]
/hold
depends on: https://github.com/openshift/assisted-service/pull/2598

**Comment by rollandf** [comment]
:+1:

File: scripts/deploy_assisted_service.sh [hunk 1]
```diff
@@ -25,15 +23,14 @@ export REGISTRY_SERVICE_PORT=80
...
-    OPENSHIFT_VERSIONS_CMD="OPENSHIFT_VERSIONS=$(printf "${json_template}" "${OPENSHIFT_VERSIONS}")"
+if [[ "${ENABLE_KUBE_API}" == "true" || "${DEPLOY_TARGET}" == "operator" ]]; then
+    # Only OS_IMAGES list is required in kube-api flow (assisted-service is using defaults if missing)
+    if [ -z "${OPENSHIFT_VERSIONS:-}" ]; then
+        export OPENSHIFT_VERSIONS='{}'
+    fi
+    if [ -z "${RELEASE_IMAGES:-}" ]; then
+        export RELEASE_IMAGES='[]'
+    fi
 fi
```
Comment (YuviGold) [inline scripts/deploy_assisted_service.sh:34]: Could this section be deleted entirely?

Comment (rollandf) [reply scripts/deploy_assisted_service.sh:34]: We want to test that dynamic version using ClusterImageSet is working, this is the reason for clearing these.

Comment (danielerez) [reply scripts/deploy_assisted_service.sh:34]: Right, it's for simulating the kube-api/operator scenario, in which only OS_IMAGES var is available.

**Pull request merged**